app/code/Transoft/Blog/Block/Adminhtml/Post/Grid.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
We are using UI component to render blog grid. Aren't we? So why we need this one?

Reply:
removed